src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -1105,14 +1122,16 @@ public static IEqualityComparer<HashSet<T>> CreateSetComparer()
         /// greater than or equal to capacity.
         /// </summary>
         /// <param name="capacity"></param>
-        private void Initialize(int capacity)
+        private int Initialize(int capacity)
         {
             Debug.Assert(_buckets == null, "Initialize was called but _buckets was non-null");
 
             int size = HashHelpers.GetPrime(capacity);
 
             _buckets = new int[size];
             _slots = new Slot[size];
+
```

Comment:
Nit: Extra line.

Reply:
I think I'm not getting the extra line logic. :) 
Sometimes I'm asked to add extra line, sometimes to remove extra line.
I thought it's fine to have at most one line in between two stateements and what's wrong is to have more than one blank line between 2 statements.

Reply:
Haha I know, it’s fine either way, if you leave it that good. That is why I added nit in this comment.